staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -155,6 +156,8 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&s.StorageConfig.CompactionInterval, "etcd-compaction-interval", s.StorageConfig.CompactionInterval,
 		"The interval of compaction requests. If 0, the compaction request from apiserver is disabled.")
 
+	fs.DurationVar(&s.StorageConfig.CountMetricPollPeriod, "etcd-count-metric-poll-period", time.Minute, ""+
```

Comment:
instead of `time.Minute` using `s.StorageConfig.CountMetricPollPeriod` allows overriding defaults like the other options above

Reply:
And you can still set a default in `NewEtcdOptions` above

Reply:
It's not perfect, but done.